schemas/affiliation.json

```diff
@@ -1,11 +1,14 @@
 {
   "$id": "https://hbz.github.io/lobid/schemas/affiliation.json",
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "title": "Affiliation",
-  "description": "School or organisation, the person is affiliated with.",
+  "title": "Affiliierung",
```

Comment:
Better is "(institutionelle) Zugehörigkeit"?

(Im Deutschen gibts zwar Affiliation, aber Affiliierung hab ich noch nie gehört und im Duden finde ich es nicht)